.travis.yml

```diff
@@ -1,4 +1,5 @@
 sudo: false
+dist: precise
```

Comment:
Any reason why we don't go for 14.04? Then we could also keep postgre 9.5 right?